src/yoroi-wallets/cardano/ShelleyWallet.ts

```diff
@@ -973,6 +987,32 @@ export class ShelleyWallet implements WalletInterface {
     return api.fetchCurrentPrice(symbol, this.getBackendConfig())
   }
 
+  async fetchNfts(): Promise<YoroiNFT[]> {
```

Comment:
I'm not really sure if this is the best place to have this function. What are your thoughts?

Reply:
this should be moved into `/cardano/api.ts`

Reply:
we can rework it later for now it's ok